packages/pools/src/router/routes.ts

```diff
@@ -150,6 +150,50 @@ export class OptimizedRoutes implements TokenOutGivenInRouter {
 
     let routes = this.getCandidateRoutes(tokenIn.denom, tokenOutDenom);
 
+    for (let i = 0; i < routes.length; i++) {
+      const curRoute = routes[i];
+      if (curRoute.pools.length == 1) {
+        const singlePool = curRoute.pools[0];
+
+        const denomSize = singlePool.poolAssetDenoms.length;
+        if (denomSize != 2) {
+          continue;
+        }
+
+        const denomA = singlePool.poolAssetDenoms[0];
+        const denomB = singlePool.poolAssetDenoms[1];
+
+        if (tokenIn.denom != denomA && tokenIn.denom != denomB) {
+          continue;
+        }
+
+        if (tokenOutDenom != denomA && tokenOutDenom != denomB) {
+          continue;
+        }
+
+        // Transmuter pools expect a 1:1 swap with no slippage.
+        // If transmuter pool is encountered that has token in and token out
+        // return it as a single pool swap.
+        const cwPoolIDs = ["1175", "1176", "1211", "1212"];
```

Comment:
Can we declare these pools as constants at the top of the page? What does `cw` stand for?

Reply:
Done. Cw stands for cosmwasm. Changed to transmuter for clarity.

Transmuter is a special case of a cosmwasm pool